Documentation/HowToGuides/UsingCombinedActions/README.md

```diff
@@ -0,0 +1,205 @@
+
+![Set Transformed Function](assets/images/SetTransformedFunction.png)
+
+### Step 15
+
+Expand the `Tilia.Input.CombinedActions.Unity` Package directory in the Unity Project window and select the `Packages -> Tilia.Input.CombinedActions.Unity -> Runtime -> Prefabs` directory then drag and drop the `Input.CombinedActions.AxesToAngle` prefab into the Unity hierarchy window.
+
+![Import Axes To Angle](assets/images/ImportAxesToAngle.png)
+
+### Step 16
+
+On the `Input.CombinedActions.AxesToAngle` GameObject, Drag and drop the `Input.CombinedActions.BooleanTo1DAxisAction Horizontal` GameObject into the `Horizontal Axis` property on the `Axis Settings` component.
+
+![Drag And Drop Axes To Angle Into Horizontal Axis](assets/images/DragAndDropAxesToAngleIntoHorizontalAxis.png)
+
+Drag and drop the `Input.CombinedActions.BooleanTo1DAxisAction Vertical` GameObject into the `Vertical` Axis` property on the `Axis Settings` component.
+
+![Drag And Drop Axes To Angle Into Vertical Axis](assets/images/DragAndDropAxesToAngleIntoVerticalAxis.png)
+
+### Step 17
+
+Select the `Input.CombinedActions.AxesToAngle` GameObject from the Unity Hierarchy and click the `+` symbol in the bottom right corner of the `Value Changed` event parameter on the `Axes To Angle Action` component.
+
+Drag and drop the `Rotation` GameObject into the event listener box that appears on the `Value Changed` event parameter on the `Axes To Angle Action` component that displays `None (Object)`.
+
+![Create Axes To Angle Event](assets/images/CreateAxesToAngleEvent.png)
+
+Select a function to perform when the `Value Changed` event is emitted. For this example, select the `Float To Vector3` -> `CurrentY` function (be sure to select `Dynamic Vector3 - CurrentY` for this example).
+
+![Set Axes Angle Function](assets/images/SetAxesAngleFunction.png)
+
+### Step 18
+
+Select the `Input.CombinedActions.AxesToAngle` GameObject from the Unity Hierarchy and click the `+` symbol in the bottom right corner of the `Value Changed` event parameter on the `Axes To Angle Action` component to add a second `Value Changed` event.
+
+Drag and drop the `Rotation` GameObject into the event listener box that appears on the `Value Changed` event parameter on the `Axes To Angle Action` component that displays `None (Object)`.
+
+![Create Second Axes To Angle Event](assets/images/CreateSecondAxesToAngleEvent.png)
+
+Select a function to perform when the `Value Changed` event is emitted. For this example, select the `Float To Vector3` -> `DoTransform()` function (be sure to select `Static Vector3 - DoTransform()` for this example).
+
+![Set Second Axes Angle Function](assets/images/SetSecondAxesAngleFunction.png)
+
+### Done
+
+Play the Unity scene, you will notice that by pressing the keys you can rotate the position in which the GameObject is facing.
```

Comment:
reword this a bit, like

```
### Done

Play the Unity scene, you will notice that by pressing the keys you can rotate the GameObject to change the direction it is facing.
```

Reply:
Also add this image:

![image](https://user-images.githubusercontent.com/1029673/76461039-09196880-63d7-11ea-82dc-0bf9c81f74df.png)